packages/mermaid/src/diagrams/team-topology/parser/teamTopology.jison

```diff
@@ -0,0 +1,45 @@
+/** mermaid
```

Comment:
Please provide feedback on this grammar file. I have not yet added directive support but that would come in the next round of work.

Reply:
I've removed support for this boilerplate from the DB.

I've raise a new PR to remove directives from the documentation: https://github.com/mermaid-js/mermaid/pull/4688